src/platform/forms-system/src/js/web-component-patterns/addressPattern.jsx

```diff
@@ -105,9 +105,13 @@ const savedAddress = {
   stateCode: '',
 };
 
-const getAddressPath = (path, index) => {
-  const addressPath = [...path].shift();
-  return index != null ? [addressPath, index] : addressPath;
```

Comment:
this code was introduced in a recent previous commit, but I realized this wasn't quite write. The new code should be correct. (currently we don't use this pattern in arrays anywhere, so this index param shouldn't have affected any live or WIP forms)